Source/Renderer/AutomaticUniforms.js

```diff
@@ -1478,6 +1478,25 @@ define([
             }
         }),
 
+         /**
+         * An automatic GLSL uniform representing the splitter position to use when rendering imagery layers with a splitter.
```

Comment:
This is just internal doc, but could you please mention the range of the value here?